logging.conf.sample

```diff
@@ -1,5 +1,5 @@
 [loggers]
-keys=root,process,seed,prune_tiles_of_interest,enqueue_tiles_of_interest,dump_tiles_of_interest,load_tiles_of_interest,wof_process_neighbourhoods,query,consume_tile_traffic,tile_status,stuck_tiles,delete_stuck_tiles,rawr_enqueue,rawr_process
+keys=root,process,seed,prune_tiles_of_interest,enqueue_tiles_of_interest,dump_tiles_of_interest,load_tiles_of_interest,wof_process_neighbourhoods,query,consume_tile_traffic,tile_status,stuck_tiles,delete_stuck_tiles,rawr_enqueue,rawr_process,rawr_seed
```

Comment:
This is slightly OT, but; is there any way to wildcard these loggers, so that we don't have to add a new section to the config every time we use a new name?

I know in production we'd want the ability to turn them on or off, up or down individually, but it seems like it should be possible to default to `INFO` when they're not listed explicitly?

Reply:
> This is slightly OT, but; is there any way to wildcard these loggers, so that we don't have to add a new section to the config every time we use a new name?

Yea, that's a good idea.

> I know in production we'd want the ability to turn them on or off, up or down individually, but it seems like it should be possible to default to INFO when they're not listed explicitly?

I think we can do it in code. What do you think of just setting the default level to INFO for all loggers, as a default parameter? Would be here:
https://github.com/tilezen/tilequeue/blob/4db723349365712d9aa6da0034196f92a285c099/tilequeue/command.py#L303

Reply:
Here's a [pr](//github.com/tilezen/tilequeue/pull/260)